src/backend/proxy/providerPreload.ts

```diff
@@ -53,7 +56,7 @@ const sendRequest = async (...args: unknown[]) => {
   if (Object.hasOwn(result, 'error')) {
     throw result[0].error.message
   }
-  return result[0].result
+  return result
```

Comment:
Is this related? Instead of a value how it will return an array, right?
Where do we declare the types for it?

Reply:
The return type is `unknown` https://eips.ethereum.org/EIPS/eip-1193#send-deprecated

`Provider.send(...args: unknown[]): unknown;`

I'm not sure why I was treating it as an array before. I believe we may have been batching these at some point but aren't any more and it was a regression? Either way, we need to return the exact value we receive from the provider which is what result is currently.

I don't think lines 56-58 are needed either. I can remove them

Reply:
also from testing, this method actually throws now with the same error instead of returning an error prop. I believe this change was made a while back but this method was never updated

Reply:
and yes it is related because it is breaking one of our browser games